src/angular/projects/spark-core-angular/package.json

```diff
@@ -1,17 +1,11 @@
 {
-  "name": "@sparkdesignsystem/spark-core-angular",
+  "name": "spark-core-angular",
   "version": "1.0.0",
-  "description": "A collection of Spark Design System components in Angular 6+",
```

Comment:
Why the removal of the description and stuff? I know scripts needed to be removed.

Reply:
must have just accepted my side, the generator doesnt make those things, i'll replace them

Reply:
Yeh, i added a bunch of them since i was getting warnings. Thought i had one in for the "repository" as well.